Packs/FortiGate/Integrations/FortiGate/FortiGate.py

```diff
@@ -103,1209 +1503,2378 @@ def create_addr_string(list_of_addr_data_dicts):
+        custom_table_builder=build_address_table,
+        outputs_prefix=ADDRESS6_MULTICAST_CONTEXT,
+        outputs_key_field="Name",
+    )
 
 
 @logger
-def delete_service_group_command():
-    context = {}
-    group_name = demisto.args().get('groupName').encode('utf-8')
+def create_firewall_address_ipv6_multicast_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """Create firewall IPv6 multicast addresses.
 
-    delete_service_group_request(group_name)
+    Args:
+        client (Client): Session to Fortigate to run API requests.
+        args (dict[str, Any]): Arguments passed down by the CLI to configure the request.
 
-    service_group_context = {
-        'Name': group_name,
-        'Deleted': True
-    }
+    Returns:
+        CommandResults: Outputs of the command that represent an entry in warroom.
+    """
+    name = args.get("name", "")
+    vdom = args.get("vdom", DEFAULT_VDOM)
+    comment = args.get("comment")
+    address = args.get("address")
+    mask = arg_to_number(args.get("mask"))
+
+    validate_mask(mask)
+    subnet = f"{address}/{mask}"
+    validate_ipv6_networks([subnet])
+
+    response = client.create_firewall_address_ipv6_multicast(
+        name=name,
+        vdom=vdom,
+        comment=comment,
+        subnet=subnet,
+    )
+    output = build_address_outputs(args)
+    readable_output = f"The firewall address multicast IPv6 '{name}' was successfully created."
+
+    return CommandResults(
+        outputs_prefix=ADDRESS6_MULTICAST_CONTEXT,
+        outputs_key_field="Name",
+        outputs=output,
+        readable_output=readable_output,
+        raw_response=response,
+    )
 
-    contents = service_group_context
-    context['Fortigate.ServiceGroup(val.Name && val.Name === obj.Name)'] = service_group_context
 
-    demisto.results({
-        'Type': entryTypes['note'],
-        'ContentsFormat': formats['json'],
-        'Contents': contents,
-        'ReadableContentsFormat': formats['markdown'],
-        'HumanReadable': tableToMarkdown('FortiGate service group: ' + group_name + ' was deleted successfully', contents),
-        'EntryContext': context
-    })
+@logger
+def update_firewall_address_ipv6_multicast_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """Update firewall IPv6 multicast addresses.
+
+    Args:
+        client (Client): Session to Fortigate to run API requests.
+        args (dict[str, Any]): Arguments passed down by the CLI to configure the request.
+
+    Raises:
+        DemistoException: If `address` and `mask` are not provided together.
+
+    Returns:
+        CommandResults: Outputs of the command that represent an entry in warroom.
+    """
+    name = args.get("name", "")
+    vdom = args.get("vdom", DEFAULT_VDOM)
+    comment = args.get("comment")
+    address = args.get("address")
+    mask = arg_to_number(args.get("mask"))
+
+    subnet = None
+    address_provided = bool(address)
+    mask_provided = mask is not None
```

Comment:
In the `elif` a `xor` logic is being used, if we just use the value of `address` it will never work, because when comparing the difference of `str` and `bool` we will always get `True`.
It is also used for the first `if` to just make the code easier to understand, here the `address` can be used if wanted.